pkg/apis/operator/v1alpha1/tektonaddon_types.go

```diff
@@ -73,6 +71,9 @@ type Addon struct {
 	// Params is the list of params passed for Addon customization
 	// +optional
 	Params []Param `json:"params,omitempty"`
+	// InstallPipelinesAsCode field defines whether to install PAC
+	// +optional
+	EnablePAC *bool `json:"enablePipelinesAsCode,omitempty"`
```

Comment:
could you add information about this field in the [TektonAddon.md](https://github.com/tektoncd/operator/blob/main/docs/TektonAddon.md#params) doc file